examples/dataset/object_detection_2d/object_detection_2d/upload_results.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from commons import defaultdict
+from typing import List
+from typing import Dict
+from typing import Any
+
+import pandas as pd
+from metrics import test_sample_metrics
+from commons import load_data
+from commons import DATASET
+from commons import BUCKET
+
+import kolena
+from kolena.dataset import upload_results
+from kolena.dataset import download_dataset
+from kolena._experimental.object_detection.utils import filter_inferences
```

Comment:
Can imports can be simplified? I'm not too sure, but `from kolena.workflow.annotation` can become `from kolena.annotation`, which is known by the information given in init files

Reply:
yeah - i ended up simplifying (the filter_inferences was giving me a ton of errors in the pre-commit so I just put the helper in the script)